### DISCUSSION TITLE
using helix in nbb

### DISCUSSION THREAD

**Discussion by schneiderlin** [babashka/nbb#354] [Q&A] [answered]
nbb.edn
```edn
{:deps {lilactown/helix {:mvn/version "0.2.1"}}}
```

.nbb/.cache/xxxx/nbb-deps contains helix.
but when I use it like

```clojurescript
(ns test
  (:require ["ink" :refer [render Text Box Spacer]]
            [helix.core :refer [defnc $]]
            [reagent.core :as r]
            [nbb.nrepl-server :as nrepl]))
```

throw `defnc does not exist`

**Comment by borkdude** [comment, marked as answer]
I don't think helix will work with nbb (as not all Clojure works with the SCI interpreter), but I could have a look as to why this is.
Btw, reagent is bundled with nbb so you can use that as well.

**Reply by borkdude** [reply]
I think this is because macros in helix are defined in `.clj` file and there is exists also a `.cljs` file. Because nbb supports macros inside of `.cljs` files, it doesn't load the `.clj` in addition to that.
